scripts/licenseChecker.js

```diff
@@ -0,0 +1,50 @@
+const checker = require('license-checker');
```

Comment:
Good call. This makes a lot of sense. My thinking is that I noticed dependency review is going to support license checking very soon, so we probably want to migrate over to that once the feature is stable since we already use it to check vulnerabilities.